package.json

```diff
@@ -4,13 +4,13 @@
   "description": "Hathor Wallet Headless, i.e., without graphical user interface",
   "main": "index.js",
   "engines": {
-    "node": ">=20.0.0",
+    "node": ">=22.0.0",
     "npm": ">=10.0.0"
   },
   "dependencies": {
     "@dinamonetworks/hsm-dinamo": "4.9.1",
     "@hathor/healthcheck-lib": "0.1.0",
-    "@hathor/wallet-lib": "1.14.0",
+    "@hathor/wallet-lib": "git://github.com/HathorNetwork/hathor-wallet-lib#glevco/fake-release",
```

Comment:
TODO: Change this to the actual released version, when we do it.

Reply:
Done in https://github.com/HathorNetwork/hathor-wallet-headless/pull/509/commits/02de65e83931e3580415cd9ca0b0228f0a1453a9